src/content/en/updates/2018/12/badging-api.md

```diff
@@ -61,54 +68,68 @@ Examples of sites that may use this API include:
+standards community. For more information, see the
+[Origin Trials Guide for Web Developers][ot-dev-guide].
+
+### Register for the origin trial {: #ot }
+
+1. [Request a token][ot-request] for your origin.
+2. Add the token to your pages, there are two ways to provide this token on
+   any pages in your origin:
+     - Add an `origin-trial` `<meta>` tag to the head of any page. For example,
+       this may look something like:
+       `<meta http-equiv="origin-trial" content="TOKEN_GOES_HERE">`
+     - If you can configure your server, you can also provide the token on pages
+       using an `Origin-Trial` HTTP header. The resulting response header should
+       look something like: `Origin-Trial: TOKEN_GOES_HERE`
+
+
+### Using the badging API during the origin trial
+
+Dogfood: During the origin trial, the API will be available via
+`window.ExperimentalBadging`. The below code is based on the current design,
+and will change before it lands in the browser as a standardized API.
 
 To use the badging API, your web app needs to meet
 [Chrome’s installability criteria](/web/fundamentals/app-install-banners/#criteria),
 and a user must add it to their home screen.
 
-The `Badge` interface is a member object on `Window` and `Worker`. It contains
+The `ExperimentalBadge` interface is a member object on `window`. It contains
 two methods:
 
-* `set([number])`: Sets the app's badge. If a value is provided, set the
-  badge to the provided value.
+
+* `set([number])`: Sets the app's badge. If a value is provided, set the badge
+  to the provided value otherwise, display a plain white dot (or other flag as
+  appropriate to the platform).
 * `clear()`: Removes app's badge.
 
 
 ```js
 // In a web page
 const unreadCount = 24;
-window.Badge.set(unreadCount);
-
-// In a service worker
-self.addEventListener('sync', () => {
-  self.Badge.set(getUnreadCount());
-});
+window.ExperimentalBadge.set(unreadCount);
 ```
 
+`ExperimentalBadge.set()` and `ExperimentalBadge.clear()` can be called from
+a foreground page, or potentially in the future, a service worker. In either
+case, it affects the whole app, not just the current page.
 
-`Badge.set()` and `Badge.clear()` can be called from either a foreground page
-or potentially a service worker. In either case, it affects the whole app, not
-just the current page.
-
-Caution: The spec and explainer currently allow for strings in the badge, but
-that is being removed. Only numbers will be permitted.
-
-In some cases, the OS may not allow the exact representation of the badge, in
-this case, the browser will attempt to provide the best representation for
-that device. For example, Android only shows a white dot, instead of the
-numeric value.
-
+In some cases, the OS may not allow the exact representation of the badge,
+in this case, the browser will attempt to provide the best representation for
+that device. For example, while the Badging API isn’t supported on Android yet,
```

Comment:
Adjusted this to mention while not on Android currently, Android only supports a flag.